src/Xamarin.Android.Build.Tasks/Tasks/AndroidCreateDebugKey.cs

```diff
@@ -13,11 +13,14 @@ public class AndroidCreateDebugKey : KeyTool
 
 		public string KeyAlgorithm { get; set; }
 
+		public string StoreType { get; set; }
+
 		string dname = "CN=Android Debug,O=Android,C=US";
 
 		public AndroidCreateDebugKey ()
 		{
 			KeyAlgorithm = "RSA";
+			StoreType = "pkcs12";
```

Comment:
This can use an "inline" property setting on line 16:

```csharp
public string StoreType { get; set; } = "pkcs12";
```

Reply:
@jonpryor should I do that for all the properties while I'm in the file?

Reply:
Sure.